web/cypress/integration/Trace/CreateAssertion.spec.ts

```diff
@@ -1,215 +1,182 @@
+      .click();
+    cy.get('[data-cy=assertion-check-operator] .ant-select-selection-item').first().should('have.text', 'Equals');
+
+    cy.get('[data-cy=add-assertion-form-add-check]').click();
+
+    cy.get('[data-cy=assertion-check-attribute]').last().type('service');
+    cy.get(`${getAttributeListId(1)} + div .ant-select-item`)
+      .first()
+      .click();
+
+    cy.get('[data-cy=assertion-check-operator]').last().click();
+    cy.get(`${getComparatorListId(1)} + div .ant-select-item`)
+      .last()
+      .click();
+    cy.get('[data-cy=assertion-check-operator] .ant-select-selection-item').last().should('have.text', 'Contains');
+
+    cy.get('[data-cy=add-assertion-form-add-check]').click();
+
+    cy.get('[data-cy=assertion-check-attribute]').last().type('duration');
+    cy.get(`${getAttributeListId(2)} + div .ant-select-item`)
+      .first()
+      .click();
+
+    cy.get('[data-cy=assertion-check-operator]').last().click();
+    cy.get(`${getComparatorListId(2)} + div .ant-select-item`)
+      .last()
+      .click();
+
+    cy.get('[data-cy=assertion-check-value]').last().type('s');
+    cy.get('[data-cy=duration]').click();
+    cy.get(`[data-cy=duration-unit-μs]`).click();
+
+    cy.get('[data-cy=assertion-form-submit-button]').click();
+
+    cy.get('[data-cy=assertion-card-list]').should('be.visible');
+    cy.get('[data-cy=assertion-card]').should('have.lengthOf', 1);
   });
 
   it('should create a basic assertion using the advanced mode', () => {
-    (async () => {
-      const testId = await createTest();
-      cy.get(`[data-cy=trace-node-database]`, {timeout: 20000}).last().click();
-      cy.get('[data-cy=add-assertion-button]').click();
-      cy.get('[data-cy=assertion-form]', {timeout: 10000}).should('be.visible');
-
-      cy.get('[data-cy=mode-selector-switch]').click();
-      cy.get('[data-cy=advanced-selector] [contenteditable]')
-        .clear()
-        .type('span[tracetest.span.type = "http"] span[tracetest.span.type = "database"]:first');
-
-      cy.get('[data-cy=assertion-check-attribute]').type('db');
-      cy.get(`${getAttributeListId(0)} + div .ant-select-item`)
-        .first()
-        .click();
-
-      cy.get('[data-cy=assertion-check-operator]').click();
-      cy.get(`${getComparatorListId(0)} + div .ant-select-item`)
-        .last()
-        .click();
-      cy.get('[data-cy=assertion-check-operator] .ant-select-selection-item').should('have.text', 'Contains');
-
-      cy.get('[data-cy=assertion-form-submit-button]').click();
-
-      cy.get('[data-cy=assertion-card-list]').should('be.visible');
-      cy.get('[data-cy=assertion-card]').should('have.lengthOf', 1);
-      deleteTest(testId);
-    })();
+    cy.get(`[data-cy=trace-node-database]`, {timeout: 20000}).last().click();
+    cy.get('[data-cy=add-assertion-button]').click();
+    cy.get('[data-cy=assertion-form]', {timeout: 10000}).should('be.visible');
+
+    // cy.get('[data-cy=mode-selector-switch]').click();
```

Comment:
Should we remove this line?

Reply:
we dont have the notion of advanced editor. do know why this is not breaking current test.